### DISCUSSION TITLE
Is it possible to try out the Go version of Fabric?

### DISCUSSION THREAD

**Discussion by krishna-santosh** [danielmiessler/Fabric#467] [Q&A] [answered]
Hi,

I saw the note in the [README](https://github.com/danielmiessler/fabric?tab=readme-ov-file#navigation), about switching Fabric to Go. This sounds great, especially with the simplicity of installation and faster performance.

 Since the maintainers have got it working, is it possible to try it out now? I couldn't find any branches related to the Go version.

**Comment by codevalve** [comment]
(Bump)

**Comment by dajoneseng** [comment]
Any update on the availability of the Go version? I managed to get the CLI version working but briefly and am now spending more time trying to work out what's wrong rather than leveraging the productivity of the tool. I think the concept behind this is a game changer and has the potential to unlock many of the frustrations with interfacing with LLMs. It would be great if it was a little more user-friendly for those not from a programming background.

**Comment by RocketRene** [comment]
I am a go developer and I would love to try it out as well, I just leaned how to package a go application for ubuntu, fedora, mac etc. 
If you need help, I am avalible..

**Comment by AO12612** [comment]
very interested in betatesting the go re-write too

**Reply by xssdoctor** [reply, marked as answer]
Still working out the kinks. Will be ready for beta testing in like a week I think

**Reply by AO12612** [reply]
will the general user-facing concepts/endpoints remain the same ?
Mills etc? I've been looking for examples or docs on how to integrate a workflow with obsidian and a web ui, but I don't want to reinvent the wheel nor troubleshoot dead code ;)

**Reply by AO12612** [reply]
@xssdoctor, any news ? as I said I'd like to help with testing and possibly with documentation

**Reply by dajoneseng** [reply]
@xssdoctor any update on this? just trying to work out if I put the effort into resolving my existing issue with the python version :)

**Comment by patelsan** [comment]
Is the work still happening on go-rewrite? Could you please share branch/ repo details?

**Reply by sleeper** [reply]
Yed, work is still happening. We're waiting for @xssdoctor to merge which is probably the last PR before a first public release.
